### PR TITLE
Device Plugins: add an info about beta graduation

### PR DESCRIPTION
**Pull request by bart0sh**

### CODE REVIEW

**Review by elezar — APPROVED** [review]
/lgtm

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -159,8 +159,9 @@ The general workflow of a device plugin includes the following steps:
 
    {{< note >}}
    The processing of the fully-qualified CDI device names by the Device Manager requires
-   the `DevicePluginCDIDevices` feature gate to be enabled. This was added as an alpha feature in
-   v1.28.
+   that the `DevicePluginCDIDevices` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+   is enabled for the kubelet and the kube-apiserver. This was added as an alpha feature in Kubernetes
```
Comment (windsonsea) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:163]:
```suggestion
   is enabled for both kubelet and kube-apiserver. This was added as an alpha feature in Kubernetes
```

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -159,8 +159,9 @@ The general workflow of a device plugin includes the following steps:
...
+   v1.28 and graduated to beta in 1.29
```
Comment (windsonsea) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:164]:
```suggestion
   v1.28 and graduated to beta in v1.29.
```

**Comment by elezar** [comment]
/remove-lgtm

Let's address @windsonsea's comments first.

**Comment by bart0sh** [comment]
@windsonsea @elezar Thank you for the review! All suggestions have been merged into the PR branch. PTAL.

**Review by elezar — APPROVED** [review]
/lgtm

**Comment by bart0sh** [comment]
/assign @SergeyKanzhelev @mrunalp @derekwaynecarr @dchen1107 
for SIG-Node approval

**Comment by bart0sh** [comment]
@pacoxu thank you for the review. Code change PR has been created as suggested: https://github.com/kubernetes/kubernetes/pull/121254, PTAL.

**Comment by derekwaynecarr** [comment]
Just noting that the related PR is blocking promotion to beta pending additional e2e tests.

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 3]
```diff
@@ -159,8 +159,8 @@ The general workflow of a device plugin includes the following steps:
    {{< note >}}
    The processing of the fully-qualified CDI device names by the Device Manager requires
    that the `DevicePluginCDIDevices` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-   is enabled for the kubelet and the kube-apiserver. This was added as an alpha feature in Kubernetes
-   v1.28.
+   is enabled for both kubelet and kube-apiserver. This was added as an alpha feature in Kubernetes
+   v1.28 and graduated to beta in v1.29.
```
Comment (klueska) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:163]:
```suggestion
   is enabled for both the kubelet and the kube-apiserver. This was added as an alpha feature in Kubernetes
   v1.28 and graduated to beta in v1.29.
```

**Comment by klueska** [comment]
/approve

/hold
For https://github.com/kubernetes/kubernetes/pull/121440

**Comment by bart0sh** [comment]
/unhold
as https://github.com/kubernetes/kubernetes/pull/121440 and https://github.com/kubernetes/kubernetes/pull/121254 have been merged

**Comment by bart0sh** [comment]
@mrunalp @SergeyKanzhelev can you review / approve?

**Comment by pacoxu** [comment]
> /assign @SergeyKanzhelev @mrunalp @derekwaynecarr @dchen1107 for SIG-Node approval

/lgtm
/assign @reylejano

**Comment by Princesso** [comment]
Hello @bart0sh 👋, gentle reminder to get your PR merged before the deadline, Tuesday 28th November 2023. Thank you!

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**